### ISSUE TITLE
Major PD overhaul

### ISSUE THREAD

**Issue by DGuinness** [New-Chance-RP/Feedback#550]
<h2>Feature you are requesting.</h2>

At the moment LSPD is completely underpowered and needs to be buffed, Non of the vehicles are suitable for pursuits E.G. handling is terrible compared to the cars available for citizens, speeds of the police vehicles are no match for some cars. 
Reports of crimes are slow, the pings for stolen vehicles are too delayed so by the time you get the call the person is long gone and also need to have a description of the vehicle that has been stolen for PD to even stand a chance of doing any investigative work. More responsive and descriptive reports from locals would enhance the RP for police. The same goes for other reports.
The new menu is clunky and can take longer to navigate, Makes everything more complicated and slow paced. 
The new cuff animation is excessive. Makes for bad Rp when someone is compliment then you just kick them to the ground and cuff them, also pulls them into an animation so can be a bit scruffy if people are trying to escape and also there is the risk of other officers being pulled into the animation which I have seen happen before and the criminal getting away. 
Personally I’m not a fan of the in city radios for PD, there is no volume control so people can be extremely loud over the radio and it’s a lot of fiddling to do if you quickly need to turn your radio off to hear people around you.
The shooting at the moment is extremely scuffed, a lot of shots being hit that don’t register and then all of a sudden you get dropped. 

<h2>What are the benefits to your feature?</h2>

I feel there is a lot of tension in PD, a lot of the tools given to us to help us catch criminals are ineffective at this time, cars can’t handle, menus are fiddly then if it ends up in a shoot out which does happen a lot it just comes down to the luck of who’s shot actually registers leaving the police force frustrated. Everyone’s rolepaly will benefit from an improvement within PD, there will be more role play from car jackings with criminals given warnings and officers having to look out for the stolen vehicles. Cops won’t be as likely to get straight to shoot out as they feel they have more chance of catching someone with other means. 

<h2>Anything to support your feature request?</h2>

**Comment by BadBritishGamer** [comment]
If you have suggestions for PD improvements they have to be agreed and gone through the PD Command ranks. Although your points are valid it doesn't change the fact they have to be the ones to approve any changes that happen.

If every officer had their 2 cents in how the PD runs it'd be impossible to maintain and develop, so to keep it easier on the devs by having it agreed to by command ranks allows for a more unified input.

**Comment by DGuinness** [comment]
Majority of this Feedback is all Dev based, e.g. vehicle handeling, menues and scripts. If police command need to be inforemed then pass it on to them as well.

**Comment by driftoooo** [comment]
Hi,
  As far as vehicle handling is concerned, this will be sorted in due course. At the moment, some civilian cars have GTA handling and others are done, making balancing pd vehicles a task. We need to make them fast enough to catch the complete cars and good enough at turning to be able to catch the GTA handling vehicles whilst keeping it fun for everyone. Once all civilian vehicles have received handling PD will receive a more realistic handling file for all police cruisers. For now, the temporary changes to the PD vehicles to balance them between GTA handling vehicles and complete vehicles should be sorted soon once we've ironed out some final issues. Feel free to message myself on discord (Driftoooo#9895) with any questions regarding the vehicle handling for PD vehicles. :)

**Comment by DGuinness** [comment]
Cheers for the response Driftooo.

**Comment by Zopfco** [comment]
If you'll excuse the peanut gallery comments from someone outside PD that's given the LEO/Crim meta a lot of thought?

The anachronistic radio & inconsistent gunplay affect everyone.  Of course PD needs capable pursuit vics but since this came up I just wanted to mention 3 things that I think will help improve the quality of LEO/criminal RP on New Chance:

1. **Prevention/Risk** - This drives people to more interesting crime than random muggings. OP mentioned local reports & that's a perfect example.  Robbing someone in a public place in broad daylight is reckless and should be punished with game mechanics.  Cameras are everywhere these days & in every pocket.  

2. **Forensics** -  I'm sure this is a big task but fleeing the scene shouldn't always mean getting away with it.  Something like a gang shootout would leave all kinds of evidence that would eventually point back to the perps.  I've seen fantastic RP come out of this before & think it makes for more interesting and better crim characters over time, and keeps PD engaged when they're late to a scene etc.

3. **Prison RP** - If going to jail sucks in the OOC sense crims are going to stick to low-risk muggings, chain robbing convenience stores out in BC & so on.  Going to prison should be just another part of playing a criminal character & provide new lines of RP, like access to certain contacts, gangs controlling different cell blocks or other things inside.  This would apply to the court system as that gets implemented.  These things can help steer new crim players into better RP.

tl;dr - LEO shouldn't be as dependent on having superior numbers online.  In character, the PD is _always_ "online".
